### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by teddyjkwang** [seragENTp/RecurrentSpatialTransformer#1]
Hi,thanks for the work.
Did you try on the dataset and how is the accuracy?

**Comment by LevinJ** [comment]
Hi , @seragENTp ,  I am thinking of  implementing that paper in tensoflow too. So am wondering if you have repeated the result in that parper? Thanks.

**Comment by lmolhw5252** [comment]
Hi, I use your code but it cannot convergence,and loss is always 2.29-2.3.
What's my problem? I don't change anymore.

**Comment by teddyjkwang** [comment]
@lmolhw5252 I found the same problem.I thought the way to define the target of cross entropy is not right but not sure

**Comment by lmolhw5252** [comment]
@SwaggyK93 Can you fix it?

**Comment by LevinJ** [comment]
Hi @SwaggyK93 , @lmolhw5252 , just thought you might be interested, I implemented the model in tensorflow and was able to reproduce the paper result.


[Here](https://github.com/LevinJ/cluttered_mnist_sequence) is my code implementation.

**Comment by teddyjkwang** [comment]
@LevinJ 
THANKs so much for that!!

**Comment by LevinJ** [comment]
顺便说一下，有一个计算机视觉方面的公开课，如果你还没上的，可以考虑下，我觉得非常不错的。

https://github.com/machinelearningnanodegree/stanford-cs231

https://www.youtube.com/watch?v=vT1JzLTH4G4&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk

**Comment by LevinJ** [comment]
不好意思，发错了。

**Comment by teddyjkwang** [comment]
哈哈谢谢老师 我会认真学习的

**Comment by lmolhw5252** [comment]
@LevinJ 非常感谢！您好我想请教您个问题，这篇论文中只是使用了3个数的MNIST数据集，并且设定切片个数为3，可是STN-OCR一文中，也是引用了这个方法，但是那篇论文好像没有切片个数的限制，请问您对这个有研究么？
“STN-OCR: A single Neural Network for Text Detection and Text Recognition”这个是那篇论文的题目。

**Comment by LevinJ** [comment]
你好，我大致看了下论文，STN-OCR也是用了固定数目的切片。 文章有提到将来可能会研究由模型自动确定切片数目（或者说textline/characters 数目）。 你可以查看一下 "5.Conclusion"部分。

**Comment by lmolhw5252** [comment]
@LevinJ 非常感谢您！！
